.github/workflows/beta.yml

```diff
@@ -14,9 +14,7 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
           scope: '@aws'
       - run: npm ci
-      - run: npm run lint
       - run: npm run build
-      - run: npm test
```

Comment:
No linting and tests on beta?

Reply:
No, not required, I want to keep the beta release strategy as fast as possible and consumer should be aware that it is an untested beta version. We may still improve it with less tests and checks but for now not necessary.